pkg/bootstrapper/bootstrapper.go

```diff
@@ -46,7 +46,7 @@ const (
 	// serviceWaitTime is an arbitrary amount of time to wait for the Windows service API to complete requests
 	serviceWaitTime = time.Second * 10
 	// certDirectory is where the kubelet will look for certificates
-	certDirectory = "c:/var/lib/kubelet/pki/"
+	certDirectory = "c:\\var\\lib\\kubelet\\pki\\"
```

Comment:
Please use `filepath.Join` here too.

Reply:
`filepath.Join` is evaluated at run time but we are keeping all the kubelet args as constants, so I think this is okay as is.

Reply:
This argument is a Windows path - the path separators we provided are for Linux based systems, so I am changing it here.